tests/pytests/test_index_oom.py

```diff
@@ -73,6 +73,15 @@ def test_stop_indexing_low_mem_verbosity(env):
   index_oom_count = env.cmd('INFO', 'modules')['search_OOM_indexing_failures_indexes_count']
   env.assertEqual(index_oom_count, 1)
 
+  # Check Verbosity of HSET after OOM
```

Comment:
Please test for update scenario and JSON index as well

Reply:
Added update scenario, JSON added in a different test